### DISCUSSION TITLE
Relation between K-size and successful proof rate

### DISCUSSION THREAD

**Discussion by SlugFiller** [Chia-Network/chia-blockchain#7672] [Q&A]
A k-35 plot takes roughly the same size as 8 k-32 plots. Does it also mean it has 8 time the odds of finding a solution to a puzzle at a given difficulty as a k-32 plot? More? Less?

**Comment by gryan315** [comment]
No, it just means when it doesn't find a proof, it does it 8 times harder. It just has more tables of numbers in it, the numbers are matched to have a mathematically significant relationship, and this creates the "proof". No plot will have more or less of a chance than another at having a proof, they all have a chance.

**Comment by MaximusChia** [comment]
What do you mean? so having a 202GB Plot have the same probability than having a k32 100Gb Plot?
Trying to say its a waste of space?

**Comment by keliew** [comment]
If you have 512 k32 plots vs 64 k35 plots, the frequency for k32 plots passing through the filter is higher. After the filter, it's all about having the proof required. Obviously, a single k35 have more hashes than a single k32, so when each passed through the filter, it's easy to guess which has a higher probability to win.

As all plots are equal and by looking from the perspective of _space_, they all seem to be practically the same.

Although, that's just part of the whole puzzle...you still have the Timelord to deal with. He/She/It may not like your plot's face.

**Reply by MaximusChia** [reply]
Weird...thats a direct disadvantage of having bigger plots.
SO For the same Drive capacity, a Drive full of K32 will have almost 2x chances of being selected than the Drive full of K33... Because the number of plots, even when they equal same total size...

**Reply by keliew** [reply]
> Weird...thats a direct disadvantage of having bigger plots.
> SO For the same Drive capacity, a Drive full of K32 will have almost 2x chances of being selected than the Drive full of K33... Because the number of plots, even when they equal same total size...

In that sense, yes.

Chia is really a puzzle game. ;)
